### PR TITLE
quest.task.ground.destroy_multi_and_loot

### PR DESCRIPTION
**Pull request by madsboddum**
Relates to #1388

The first commit contains a big SDB cleanup. When the SDBs were re-generated using CU clientdata the first time, some NGE ones were still in there. I simply deleted everything and re-generated, so we only have the CU questtasks now.
I split it up so viewing the important changes in the second commit would be easier.

### CODE REVIEW

File: src/utility/java/com/projectswg/utility/ClientdataConvertSpecific.java [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+ * along with Holocore.  If not, see <http://www.gnu.org/licenses/>.               *
+ ***********************************************************************************/
 package com.projectswg.utility;
 
 import com.projectswg.utility.clientdata.Converters;
 
 public class ClientdataConvertSpecific {
 	
 	public static void main(String [] args) {
-		Converters.OBJECTS_BUILDING_CELLS.load();
```
Comment (Undercova) [inline src/utility/java/com/projectswg/utility/ClientdataConvertSpecific.java:8]: is this removal correct ?

Comment (madsboddum) [reply src/utility/java/com/projectswg/utility/ClientdataConvertSpecific.java:8]: Nope, that got lost on me with the massive amount of files. I will revert this change.

**Comment by Undercova** [comment]
I have tested this locally and it works

**Comment by madsboddum** [comment]
Thanks!
________________________________

**Pull request merged**